src/googleIt.js

```diff
@@ -36,14 +36,23 @@ export function openInBrowser(open, results) {
 }
 
 export function getSnippet(elem) {
-  return elem.children
-    .map((child) => {
-      if (!child.data) {
-        return child.children.map((c) => c.data);
-      }
-      return child.data;
-    })
-    .join('');
+  // recursive function to get "all" the returned data from Google
+  function findData(child) {
+    if (!child.data) {
```

Comment:
Maybe a bit cleaner `return c.data || findData(c)`?

Reply:
@IlCallo  I've had to make changes because of rstrictiove eslint - please review update if you can. Thanks.

Reply:
Sorry, should have read that beeter. Yes, you are right.

Reply:
@IlCallo You should see the requested changes now. I always appreciate your reviews. Thanks.